### PR TITLE
[navermaps] Update types of MarkerOptions

### PR DESCRIPTION
**Pull request by SuhwanCha**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://navermaps.github.io/maps.js.ncp/docs/naver.maps.Marker.html#toc39__anchor
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

This PR updates types of MarkerOptions to fit document.

### CODE REVIEW

File: types/navermaps/index.d.ts [hunk 1]
```diff
@@ -133,11 +133,11 @@ declare namespace naver.maps {
         zoomOrigin?: any;
     }
     interface MarkerOptions {
-        animation?: any;
-        map?: Map | undefined;
-        position?: any;
-        icon?: any;
-        shape?: any;
+        animation?: Animation;
+        map: Map | undefined;
```
Comment (MysteryBlokHed) [inline types/navermaps/index.d.ts:137]: If this value is mandatory, the `undefined` option should probably be removed:

```suggestion
        map: Map;
```

File: types/navermaps/index.d.ts [hunk 2]
```diff
@@ -133,11 +133,11 @@ declare namespace naver.maps {
...
+        position: any;
```
Comment (MysteryBlokHed) [inline types/navermaps/index.d.ts:138]: This type can probably be updated as well:

```suggestion
        position: Coord | CoordLiteral;
```

**Review by MysteryBlokHed — CHANGES_REQUESTED** [review]
A couple of comments here, but everything else looks good 😄

File: types/navermaps/index.d.ts (hunk 2 above)
Comment (SuhwanCha) [reply types/navermaps/index.d.ts:138]: My mistake, thank you for reviewing! 😊

**Review by MysteryBlokHed — APPROVED** [review]
LGTM!

**Comment by SuhwanCha** [comment]
Ready to merge

**Pull request merged**